src/qibocal/cli/builders.py

```diff
@@ -240,7 +240,7 @@ def _run(self, mode: ExecutionMode):
             timing_task["tot"] = timing_task["acquisition"] + result_time
             self.meta[task_id] = timing_task
             # TODO: reactivate iterative dump
-            # self.dump_report()
+            self.dump_report()
```

Comment:
I remember why I commented this out.
It fails when you run more than one action.
Until I solve it in #406 it is better to keep it commented.
```suggestion
            # self.dump_report()
```